### PR TITLE
Feature/466 copy to clipboard btn - closes #466

### LINKED ISSUE #466: Add a "Copy" button for API key
Figure out how make copying the API key to clipboard easier than having to highlight the key and then press Ctrl+C or use other methods for copying (context menu etc.) One option would be adding a "Copy" button next to API key on the Profile page.

### PR DESCRIPTION
**Pull request by frenchbread**

### CODE REVIEW

**Comment by frenchbread** [comment]
@apinf/developers Please review. #466 is related.

File: client/views/profile/profile.js [hunk 1]
```diff
@@ -8,3 +8,31 @@ AutoForm.hooks({
     }
   }
 });
+
+Template.profile.rendered = function () {
+
+  // initializes button
+  var copyButton = $("<a class=\"btn btn-default btn-xs\" id=\"copyApi\"> Copy API to clipboard</a>");
```
Comment (brylie) [inline client/views/profile/profile.js]: Why are we creating/appending the copy button in the JS? Is it conditionally loaded?

File: client/views/profile/profile.less [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#copyApi{
+  margin-top:10px;
```
Comment (brylie) [inline client/views/profile/profile.less]: Please use relative values; em or rem.

**Comment by frenchbread** [comment]
@elnzv Thanks for fix.

**Pull request merged**